subsys/bluetooth/controller/ll_sw/ull_iso.c

```diff
@@ -228,6 +247,56 @@ uint8_t ll_setup_iso_path(uint16_t handle, uint8_t path_dir, uint8_t path_id,
 	}
 #endif
 
+#if defined(CONFIG_BT_CTLR_SYNC_ISO)
+	struct lll_sync_iso_stream *stream;
+	struct ll_sync_iso_set *sync_iso;
+	isoal_sink_handle_t sink_handle;
+	struct lll_sync_iso *lll_iso;
+	struct ll_iso_datapath *dp;
+	uint32_t sdu_interval;
+	uint16_t iso_interval;
+	uint8_t burst_number;
+	int err;
+
+	if (path_dir != BT_HCI_DATAPATH_DIR_CTLR_TO_HOST) {
+		return 0;
+	}
+
+	/* Allocate and configure datapath */
+	dp = mem_acquire(&datapath_free);
+	if (!dp) {
+		return BT_HCI_ERR_CMD_DISALLOWED;
+	}
+
+	dp->path_dir      = path_dir;
+	dp->path_id       = path_id;
+	dp->coding_format = coding_format;
+	dp->company_id    = company_id;
+
+	/* TODO dp->sync_delay    = controller_delay; ?*/
+
+	sync_iso = ull_sync_iso_by_stream_get(handle);
+	lll_iso = &sync_iso->lll;
+
+	burst_number = lll_iso->bn;
+	sdu_interval = lll_iso->sdu_interval;
+	iso_interval = lll_iso->iso_interval;
+
+	err = isoal_sink_create(&sink_handle, handle, burst_number,
+				sdu_interval, iso_interval, sink_sdu_alloc_hci,
+				sink_sdu_emit_hci, sink_sdu_write_hci);
+	if (err) {
+		return BT_HCI_ERR_CMD_DISALLOWED;
```

Comment:
Will it always be `BT_HCI_ERR_CMD_DISALLOWED`? Never `BT_HCI_ERR_INVALID_PARAM`?